formats/hocon/src/main/kotlin/kotlinx/serialization/hocon/HoconEncoder.kt

```diff
@@ -7,17 +7,59 @@ package kotlinx.serialization.hocon
 import com.typesafe.config.*
 import kotlin.time.*
 import kotlinx.serialization.*
-import kotlinx.serialization.builtins.serializer
 import kotlinx.serialization.descriptors.*
 import kotlinx.serialization.encoding.*
+import kotlinx.serialization.hocon.internal.*
 import kotlinx.serialization.internal.*
 import kotlinx.serialization.modules.*
 
+/**
+ * Encoder used by Hocon during serialization.
+ * This interface allows to call methods from the Lightbend/config library on the [Config] object.
```

Comment:
This line seems to be copied from the decoder and simply is not true.
WDYT of "This interface allows intercepting serialization process and insertion of arbitrary [ConfigValue] into the output"?

Reply:
Yes, that's what I meant